Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -190,9 +207,12 @@ extension SubscriptionsViewController {
         guard let user = AuthManager.currentUser() else { return }
         guard let labelUsername = self.labelUsername else { return }
         guard let viewUserStatus = self.viewUserStatus else { return }
+        guard let avatarView = self.avatarView else { return }
 
         labelUsername.text = user.displayName()
-
+        avatarView.user = user
+        avatarView.frame = CGRect(x: 0, y: 0, width: avatarViewContainer.frame.size.width, height: avatarViewContainer.frame.size.height)
```

Comment:
We can move this line to the `didSet {}` method and remove it from here. This method is suppose to only change data.

Reply:
I had tried to leave it in the didset, but at this point I get nil, I'm going to see where I can put it so I've already rendered the container on the screen.

Reply:
I leaved a comment about why is nil... you need to user only `avatarView` instead of `self.avatarView` in that line... you don't have it in your self. reference yet.